babel.config.mjs

```diff
@@ -18,6 +18,7 @@ import { findUp } from "find-up";
 import { readFile } from "fs/promises";
 import * as path from "node:path";
 
+process.env.STABLE_PACKAGE_CLIENT_VERSION = "~2.0.11";
```

Comment:
After we move this to the other repo, we can put a devDependency on `@osdk/client` in the packages that need this and can just read from there instead of doing the hard coding.